src/Check/ExtensionsDuplicate.php

```diff
@@ -86,7 +86,7 @@ public function getResultWarn() {
     else {
       foreach ($this->registry->extensions_dupe as $name => $extension_infos) {
         $ret_val .= PHP_EOL;
-        if (!drush_get_option('json')) {
+        if (true /* !drush_get_option('json') */) {
```

Comment:
Is this debugging that should be removed, or should the if statement just be taken out period?

Reply:
This is code that was never ported for Drush 9+

The old code calls `drush_get_option`, which is always a no-op in Drush 9, and is a "function not found" in Drush 10.

These classes have no access to the commandline options in local variables or fields; the original code relied on global state to optimize the output formatting in json mode. The work to refactor to get a few spaces out of the output is still tbd.

A better comment here would be helpful.